### PR TITLE
Use one1 address for staking rpcs.

### PR DESCRIPTION
**Pull request by rlan35**
tested locally.

### CODE REVIEW

File: internal/hmyapi/types.go [hunk 1]
```diff
@@ -6,6 +6,8 @@ import (
 	"strings"
 	"time"
 
+	"github.com/harmony-one/go-sdk/pkg/address"
```
Comment (fxfactorial) [inline internal/hmyapi/types.go]: wrong import - this will give go-sdk cyclic imports problem

Comment (rlan35) [reply internal/hmyapi/types.go]: yes, corrected.

**Comment by rlan35** [comment]
> Travis failed. Better to use internal/common as internal_common, and methods from there rather than address2 from go-sdk/pkg/address. As less as possible outside of repo libs usage.

good point. corrected.

**Comment by rlan35** [comment]
jenkins, test this please.

**Pull request merged**